src/internal/connector/graph/concurrency_middleware.go

```diff
@@ -149,6 +149,10 @@ const (
 	SingleGetOrDeltaLC = 1
 	// limit consumption rate for anything permissions related
 	PermissionsLC = 5
+	// https://learn.microsoft.com/en-us/sharepoint/dev/general-development
+	// /how-to-avoid-getting-throttled-or-blocked-in-sharepoint-online#application-throttling
+	// Any create, delete, update or upload operation take 2 tokens
+	CreateLC = 2
```

Comment:
These cases should already fall under the `driveDefaultLC`.

Reply:
Hmm, I didn't think about that. That said, I would like to keep it just to be explicit.

Reply:
It's a nit, but I'd rather it get dropped.  One of two patterns works here: a default value is assumed and only deviation needs to be explicit (what we currently have); or everything needs to be explicit.  This change sits at a halfway point where we have a default, and deviations, _and_ certain places where we call out the same consumption as the default.  That third condition introduces more confusion than it solves.  It implies that either the default might change while this value needs to be protected (doubtful to ever happen), or that cases which don't declare a value are incorrectly configured (but they aren't).

If you feel more comfortable about the implementation if we always declare the token consumption, that's okay.  In that case I'd ask that you ensure that declaration is complete throughout the repo.  Otherwise, things are cleaner without the one-off.